plugins/interactive/shellclient/shellclient.go

```diff
@@ -314,7 +344,8 @@ func (s *shell) Abort() error {
 	return s.err
 }
 
-func (s *shell) Wait() (bool, error) {
+// Wait will wait for remote shell to terminate and success, and error if any.
```

Comment:
nit: "Wait will wait for the remote shell to finish, by either succeeding or returning an error."